### ISSUE TITLE
Build error- not resolvable parent POM

### ISSUE THREAD

**Issue by gatemezing** [afs/lizard#1]
Hi @afs , I've just tried to follow the building steps in the readme file. I am confused by the steps 2-4. I've successfully built Mantis library in a different folder. Then, when building lizard, I get the following error: `mvn clean install
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for org.seaborne.lizard:lizard:0.0.0-SNAPSHOT: Could not find artifact org.seaborne.mantis:mantis:pom:0.0.0-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 31, column 10
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project org.seaborne.lizard:lizard:0.0.0-SNAPSHOT (/Users/gatemezing/research/lizard/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for org.seaborne.lizard:lizard:0.0.0-SNAPSHOT: Could not find artifact org.seaborne.mantis:mantis:pom:0.0.0-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 31, column 10 -> [Help 2]`.

Am I missing something in the process? TIA

**Comment by afs** [comment]
It needs to be built in conjunction with Mantis - you may be able to use the Mantis 0.2.0 released (it's in Maven central) by updating the Mantis dependency but I haven't verified that. While I get Mantis/TDB2 sorted out, I have not been keeping Lizard up-to-date.

**Comment by gatemezing** [comment]
Thanks @afs. I am still struggling here to get it work. Anyway, do you think I can just test Mantis? To put you in context, a manager with power decisions at Publication Office listened your presentation about Lizard, and asked me to see how it works  :)

**Comment by afs** [comment]
Yes - mantis is in Maven central - http://central.maven.org/maven2/org/seaborne/mantis/

Or build from source. https://github.com/afs/mantis - has notes.

Plan is that this is to use the maven release using org.seaborne as a beta release and when it seems stable enough, donate to Apache Jena.  Need to be able to release when needed at the moment.

Includes an integrated Fuseki build.

Feel free to ask questions on GH or on the Jena developer list : dev@jena.apache.org 

So far, nothing bad has been reported by any beta users.

**Comment by gatemezing** [comment]
Thanks. I hope to resolve these errors from Maven compilation :
`The following artifacts could not be resolved: org.apache.jena:jena-arq:jar:3.1.1-SNAPSHOT, org.apache.jena:jena-base:jar:tests:3.1.1-SNAPSHOT, org.apache.jena:jena-core:jar:tests:3.1.1-SNAPSHOT, org.apache.jena:jena-arq:jar:tests:3.1.1-SNAPSHOT: Could not find artifact org.apache.jena:jena-arq:jar:3.1.1-SNAPSHOT in apache-repository-snapshots (https://repository.apache.org/snapshots)`

**Comment by afs** [comment]
You can get Mantis pre-build from central.maven.org, v0.2.0.

The code on github uses Jena 3.2.0 so I don't see where "3.1.1-SNAPSHOT" comes from.

Or do you mean Lizard?

I had email from Marc and I explained the Lizard is still experimental and development is paused while I spend time on Mantis (and the $-job).  I've sent you that part of the conversation.

Lizard will need updating for things that have changed in Mantis (small things but interface changes) even is the version is set right (3.2.0).

It's not in a state for the type of evaluation that you are trying to do if I understand the objective correctly. It's not-production ready.

**Comment by gatemezing** [comment]
Yes, I meant Lizard. Sorry for the confusion.
Thanks for the details explanation. So I can just "play" with Mantis in the meantime?

**Comment by afs** [comment]
Yes - Mantis is released (v0.2.0).
